addons/mail/static/src/core/common/thread.scss

```diff
@@ -30,3 +25,10 @@
         display: none!important;
     }
 }
+
+.o-mail-Thread-banner {
+    z-index: $o-mail-NavigableList-zIndex - 1;
+    &:hover {
+        filter: brightness(98%);
+    }
```

Comment:
Out-of-scope of PR but would be nice to have a single classname to apply on all elements that we expect to have an usual hover feedback